pywbem/_listener.py

```diff
@@ -762,31 +781,14 @@ def logger(self):
         """
         :class:`py:logging.Logger`: Logger object for this listener.
 
-        Each listener object has its own separate logger object that is
-        created via :func:`py:logging.getLogger`.
+        Each listener object has its own separate logger object with the name:
 
-        The name of this logger object is:
+          `'pywbem.listener.{id}'`
 
-          ``pywbem.listener.{id}``
+        where `{id}` is a unique string for each listener object.
```

Comment:
We should probably define the scope of unique. Process, global, etc.

Reply:
DONE, changed to: "a string that is unique for each :class:`~pywbem.WBEMListener`
object within the Python process."